app/templates/ratemypdf_stats.html

```diff
@@ -204,27 +207,65 @@ <h2 class="text-center mt-5 mb-4">Estimated Form Completion Time
             </div>
 
             <div class="col-md-4">
-              {{ score_position_widget("word-count", "Total word count", 'Users <a href="https://www.nngroup.com/articles/how-little-do-users-read/">read as little as 20% of the content</a> on a longer page. Try to keep word count around 100 words.', word_count, 110, 30) }}
+              {{ score_position_widget("word-count", "Words per page", 'Users <a href="https://www.nngroup.com/articles/how-little-do-users-read/">read as little as 20% of the content</a> on a longer page. Try to keep word count around 100 words.', word_count_per_page, 110, 30) }}
             </div>
 
             <div class="col-md-4">
               {{ score_position_widget("sentence-count", "Sentences per page", "", stats.get("sentences per page"), metric_means["sentences per page"], metric_stddev["sentences per page"])}}
             </div>
 
-            <div class="col-md-4">
-              {{ percent_widget("slot-in", "Slot-in percent", '"Slot-in" fields can be answered with information the litigant knows well. An example is the litigant\'s name or address.', stats.get("slotin percent"), 70, 20) }}
-            </div>
-            <div class="col-md-4">
-              {{ percent_widget("gathered", "Gathered percent", '"Gathered" fields can be answered with information the litigant can read on a nearby piece of paper or lookup quickly. An example is the litigant\'s case number.', stats.get("gathered percent"), 10, 20) }}
-            </div>
-            <div class="col-md-4">
-              {{ percent_widget("third-party", "Third party percent", '"Third party" fields can be answered only by asking someone else for the information.', stats.get("third party percent"), 10, 20) }}
-            </div>
-            <div class="col-md-4">
-              {{ percent_widget("created", "Created percent", '"Created" fields require the litigant to apply a rule to their own facts, recall a large number of facts and put them into a narrative, or otherwise create a new answer.', stats.get("created percent"), 10, 20) }}
-            </div>
+          </div>
+  <!-- Legend -->
+  <div class="d-flex mt-3">
+    <div class="legend-item">
+      <div class="legend-color bg-success"></div>
+      <span>Green: 1 standard deviation below the mean</span>
+    </div>
+    <div class="legend-item">
+      <div class="legend-color bg-warning"></div>
+      <span>Yellow: within 1 standard deviation of the mean</span>
+    </div>
+    <div class="legend-item">
+      <div class="legend-color bg-danger"></div>
+      <span>Red: more than 1 standard deviation above the mean</span>
+    </div>
+  </div>
+</div>
 
           </div>
+<!-- -->
+<div class="container">
+  <h2 class="mt-5 mb-4">Field classification</h1>
+
+  <!-- Stacked bar chart -->
+  <div class="bar w-50">
```

Comment:
If we're restricting it to 50% width, we should center it. It looks kinda off, and without clear borders at the end of the bar, it sorta looks like we're only detecting half of the field types.

Reply:
Not a big fan of the ragged edge you get with centered content in general but I can play around with borders and maybe ultimately centering will be correct for this one.

Reply:
clearer borders would be good enough IMO. Rounding the edges a bit might work well, without an explicit border